### DISCUSSION TITLE
Is there a way to define the scroll position ?

### DISCUSSION THREAD

**Discussion by sdespont** [PHPOffice/PhpSpreadsheet#3455] [Q&A] [answered]
Is there a way to define the scroll position ? I have only found a way to display or not the scrollbar.

Thanks

**Comment by MarkBaker** [comment]
I'm not even sure exactly what you mean by "scroll position".

**Comment by sdespont** [comment]
@MarkBaker Thank you for your answer.

After generating an Excel file from a template, I always set cell A1 for all worksheets

        // Set selected cell A1
        for ($i=0 ; $i<$phpExcel->getSheetCount() ; $i++) {
            $phpExcel->setActiveSheetIndex($i)
                ->setSelectedCell($cell)
            ;
        }

But if the template has been saved with a specific scoll position, when opening the file the result is like this:

![image](https://user-images.githubusercontent.com/646632/226122582-5174a261-8a3f-41a7-8940-d08261dfe0f5.png)

And it would be great if the vertical scroll would be set at the top like this
![image](https://user-images.githubusercontent.com/646632/226122677-a87ae5a2-1754-43bb-b528-728fc112dfa8.png)

**Comment by MarkBaker** [comment, marked as answer]
```php
$worksheet->setTopLeftCell('A1');
```

**Reply by sdespont** [reply]
Amazing ! Thanks :o)
